contribute/style-guides/e2e.md

```diff
@@ -150,6 +150,15 @@ describe('List test', () => {
 
 ## Debugging PhantomJS image rendering
 
+### Common Error
```

Comment:
Thanks for adding this! This is very helpful, already trying it against https://github.com/grafana/grafana/pull/21587